.gitignore

```diff
@@ -34,6 +34,7 @@ yarn-error.log*
 client/config.json
 !client/assets/
 client/assets/*
+!client/assets/**.svg
```

Comment:
are you sure it wouldn't better to explicitly specify the eye icons? Otherwise don't we risk someone to commit their own SVG files into this repo?